src/frontend/components/SøknadsSteg/2-VelgBarn/LeggTilBarn/NyttBarnKort.tsx

```diff
@@ -0,0 +1,225 @@
+import React, { useEffect, useState } from 'react';
+
+import { useIntl } from 'react-intl';
+import styled from 'styled-components/macro';
+
+import AlertStripe from 'nav-frontend-alertstriper';
+import navFarger from 'nav-frontend-core';
+import { Knapp } from 'nav-frontend-knapper';
+import Lenke from 'nav-frontend-lenker';
+import Modal from 'nav-frontend-modal';
+import { Checkbox, Input, SkjemaGruppe } from 'nav-frontend-skjema';
+import { Ingress, Systemtittel } from 'nav-frontend-typografi';
+
+import { ESvar } from '@navikt/familie-form-elements';
+
+import { useApp } from '../../../../context/AppContext';
+import { device } from '../../../../Theme';
+import JaNeiSpm from '../../../Felleskomponenter/JaNeiSpm/JaNeiSpm';
+import SpråkTekst from '../../../Felleskomponenter/SpråkTekst/SpråkTekst';
+import { useLeggTilBarn } from './useLeggTilBarn';
+
+const LeggTilBarnKortWrapper = styled.div`
+    width: 17.25rem;
+    padding: 0.625rem;
+`;
+const Informasjonsboks = styled.div`
+    padding: 2rem;
+    border-radius: 0.3rem;
+    background-color: ${navFarger.navLysGra};
+`;
+
+const StyledIngress = styled(Ingress)`
+    && {
+        font-size: 1rem;
+        font-weight: 600;
+    }
+`;
+
+const StyledSystemtittel = styled(Systemtittel)`
+    text-align: center;
+    padding: 2rem;
+`;
+
+const StyledKnapp = styled(Knapp)`
+    margin-top: 1rem;
+`;
+
+const StyledKnappIModal = styled(Knapp)`
+    margin-top: 4rem;
+`;
+
+const StyledModal = styled(Modal)`
+    && {
+        padding: 2rem;
+    }
+    width: 45rem;
+    min-height: 35rem;
+    @media all and ${device.mobile} {
+        width: 95%;
+    }
+`;
+
+const StyledInput = styled(Input)`
+    @media all and ${device.tablet} {
+        max-width: 30rem;
+    }
+`;
+
+const StyledLenke = styled(Lenke)`
+    display: block;
+    margin-top: 1rem;
+`;
+
+export const NyttBarnKort: React.FC = () => {
+    const [modalÅpen, settModalÅpen] = useState<boolean>(false);
+    const {
+        skjema,
+        validerFelterOgVisFeilmelding,
+        nullstillSkjema,
+        valideringErOk,
+    } = useLeggTilBarn();
+    const { ident, navn, fødselsdato, borMedSøker, navnUbestemt } = skjema.felter;
+    const intl = useIntl();
+    const { søknad, settSøknad } = useApp();
+
+    const submit = () => {
```

Comment:
Kanskje denne funksjonen kunne bli flyttet til useLeggTilBarn? Så får vi samlet businesslogikk på et sted :)

Reply:
Jepp, on it